UnitySetup/UnitySetup.psm1

```diff
@@ -489,9 +489,10 @@ function Install-UnitySetupInstance {
                 }
                 else { 
                     Write-Verbose "$(Get-Date): Succeeded."
+                    [Environment]::SetEnvironmentVariable("UNITY_"+$version, $destPath, [System.EnvironmentVariableTarget]::User)
```

Comment:
nit: you can put $version inside the quotes like so "UNITY_$version", no need to change this for approval